From 84f048dbb55508aeae4fe1959e46551af3f1e99b Mon Sep 17 00:00:00 2001
From: "xie.kunming" <xie.kunming@astute-tec.com>
Date: Thu, 12 Jul 2018 09:36:47 +0800
Subject: [PATCH 1/1]     patch 8.0.0749: some unicode digraphs are hard to
 remember

    Problem:    Some unicode digraphs are hard to remember.
    Solution:   Add alternatives with a backtick. (Chris Harding, closes #1861)
    https://github.com/vim/vim/commit/816e7660e1efb918ad85b5c78d21b957d1bcad17
---
 src/nvim/digraph.c | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/nvim/digraph.c b/src/nvim/digraph.c
index 218a3f060..32f298cba 100644
--- a/src/nvim/digraph.c
+++ b/src/nvim/digraph.c
@@ -1359,6 +1359,12 @@ static digr_T digraphdefault[] =
   { 'f', 't', 0xfb05 },
   { 's', 't', 0xfb06 },
 
+  // extra alternatives, easier to remember
+  {'W', '`', 0x1e80},
+  {'w', '`', 0x1e81},
+  {'Y', '`', 0x1ef2},
+  {'y', '`', 0x1ef3},
+
   // Vim 5.x compatible digraphs that don't conflict with the above
   { '~', '!', 161 },  // ¡
   { 'c', '|', 162 },  // ¢
-- 
2.14.2.windows.2

